src/pages/Instructor/Styled.Instructor.js

```diff
@@ -0,0 +1,83 @@
+import styled from 'styled-components';
+import variables from '../../styles/variable';
+
+const COLOR = ['skyblue', 'red', 'yellow'];
+
+const S = {};
+
+S.InstructorTitle = styled.div`
+  font-size: ${({ theme }) => theme.bigFont};
+  font-weight: 800;
+  text-align: center;
+`;
+
+S.SubTitle = styled.div`
+  margin: 5rem 0;
+  font-size: ${({ theme }) => theme.smallFont};
+  font-weight: 800;
+  text-align: center;
+`;
```

Comment:
```suggestion
export const InstructorTitle = styled.div`
  font-size: ${({ theme }) => theme.bigFont};
  font-weight: 800;
  text-align: center;
`;

export const SubTitle = styled.div`
  margin: 5rem 0;
  font-size: ${({ theme }) => theme.smallFont};
  font-weight: 800;
  text-align: center;
`;
```
사의 분리에 따른 아주 좋은 접근입니다! 다만 조금더 가독성이 좋은 방법을 알려드릴테니 참고하세요!
수정함에 따라 import 방식도 변경될텐데, 그 부분도 리뷰로 남겨두었으니 참고해주세요!

Reply:
네 이렇게 사용하면 import를 그만큼 더 써야할 것 같았습니다. 참고하겠습니다.